### ISSUE TITLE
A2C performace on Seaquest

### ISSUE THREAD

**Issue by gautamb85** [ikostrikov/pytorch-a2c-ppo-acktr#53]
Hi,

I am having trouble reproducing the performance you got on Seaquest with A2C. It gets stuck pretty low (~300 points) after 10m frames. I am using the default learning rate/ settings.
Any tips to improve performance?

Thanks

**Comment by ikostrikov** [comment]
Hi,

I was able to reproduce the results.

What version of PyTorch are you using?

**Comment by gautamb85** [comment]
Hi,

Pytorch 0.3.
It works well in pong.
Are you changing / annealing the learning rate? And are you training for
10m steps or more?
I get better performace (2000 approx. ) with acktr but still not as good as
your plot.

Maybe I will clone the repo again and try.
Ps. Thanks for the amazingly well written code.

Thanks

**Comment by ikostrikov** [comment]
I just run using 

python main.py --env-name "SeaquestNoFrameskip-v4"

with no modifications.

**Comment by gautamb85** [comment]
Hi, 
I got it to work. Not really sure what went wrong in my previous run.
closing issue
